src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -82,6 +86,28 @@ private String getSanitizedConfig() {
   public void start(Promise<Void> startFuture) {
     log.debug("Startup configuration: {}", this::getSanitizedConfig);
 
+    HttpServer httpServer = vertx.createHttpServer();
+
+    httpServer.requestHandler(request -> {
+      log.info("path : {}",request.path());
```

Comment:
This log message has to be improved and the type should be debug. The frequency of the admin/health may bulge up the logs.